paper.tex

```diff
@@ -526,15 +528,30 @@ \section{Population genetics model details}\label{sec:appendix-pop-gen-model}
 \krtcomment{We need to revisit the paragraph(s) below.  We do not assume individuals are exchangeable, because we do not
 	require that all individuals have the same fitness. See, e.g., p 60 of Wakeley for why fitness == exchangeability
 	and why that is required for the standard coalescent.}
-
-Within a population, individuals are assumed to be exchangeable, but there are
-also parameters for nonrandom mating, such as selfing or cloning rates, which
-give the probability that offspring are generated from one generation to the
-next by self-fertilisation or cloning of an individual. Cloning and selfing
-rates can take any value between zero and one, and if both are specified then
-the selfing rate is the probability an offspring is generated through
-self-fertilisation given it was produced sexually (i.e. with probability one
-minus the cloning rate). We omit any details regarding selection.
+\mhcomment{The `exchangeability' assumption also breaks down under 
+	different reproductive modes, since the sampling of the alleles affects coalescent times. 
+	For example, in a highly selfing species the coalescent time for two different alleles 
+	within an individual is much lower than two alleles from different individuals. I've tried 
+	to rewrite the below paragraph to update and clarify the reproduction parameters in 
+	line with the discussions on Github. Finally, 'rates' imply events occur as Poisson-like 
```

Comment:
I agree, and think it's a subtlety of discrete vs continuous time methods (and demes supports both). The same thing comes up with migration rates, where in the continuous case it's the rate of lineages moving between demes at a given rate, and in the discrete case it's the probability that a lineage had its parent in a given deme the generation before. We'll also want to make this interpretation for discrete models really clear in the text.

Reply:
Agree here -- we need to be very clear about this in the paper.